build.module

```diff
@@ -13,7 +13,8 @@
     },
     "disabled": {
       "board": {
-        "rp2": ["adc"],
+        "host-linux": ["adc", "ble", "dgram", "gpio", "i2c", "pwm", "spi", "uart"],
```

Comment:
This section is `board`. I think you mean x86-linux by `host-linux`. If so, what about using `x86-linux`?

Reply:
Thanks for the review. It looks better so I changed it. I used the term because it's been commonly used through our build and travis configurations.